src/main/java/com/epam/training/array/Array.java

```diff
@@ -5,4 +5,36 @@
   
   private int[] items;
 
+  public Array(int[] items) {
+    this.items = items;
+  }
+
+  public int[] getItems() {
+    return items;
+  }
+
+  public void setItems(int[] items) {
+    this.items = items;
+  }
+  @Override
+  public boolean equals(Object object) {
+    if (this == object) return true;
+    if (object == null || getClass() != object.getClass()) return false;
+    //if (!super.equals(object)) return false;
```

Comment:
@Lipskaya Could you remove extra comments please

Reply:
I fixed it